multi-educator/src/Dscp/MultiEducator/Launcher/Mode.hs

```diff
@@ -155,17 +180,35 @@ loadEducator login mpassphrase = do
         newCfg = (finaliseDeferredUnsafe mempty) -- not great but it actually works
             & sub #core .~ multiEducatorConfig ^. sub #core
             & sub #witness .~ multiEducatorConfig ^. sub #witness
-            & sub #educator . sub #db .~ multiEducatorConfig ^. sub #educator . sub #db
+            & sub #educator . sub #db .~ multiEducatorSub ^. sub #db
             & sub #educator . sub #keys . sub #keyParams .~ keyParams
             & sub #educator . sub #api . option #educatorAPINoAuth .~
                 error "Do not touch, multi-educator has already run the server"
-            & sub #educator . sub #publishing .~ multiEducatorConfig ^. sub #educator . sub #publishing
+            & sub #educator . sub #publishing .~ multiEducatorSub ^. sub #publishing
+            & sub #educator . sub #certificates . option #latex .~
+                multiEducatorSub ^. sub #certificates . option #latex
+            & sub #educator . sub #certificates . option #resources .~
+                multiEducatorSub ^. sub #certificates . option #resources
+            & sub #educator . sub #certificates . sub #issuer . option #name .~
+                error "Should not use certificate config, resource is made by multi-educator"
+            & sub #educator . sub #certificates . sub #issuer . option #url .~
+                error "Should not use certificate config, resource is made by multi-educator"
```

Comment:
We could get rid of both these new errors (among other things) if we let the Educator have the option of a third-party provided issuer info. Right now this is handled by the MultiEducator alone.

It does not have any use now, but it could later on and I'd just need to change the config a but and add a CLI option